modules/cnf-configuring-fifo-priority-scheduling-for-ptp.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+//
+// * networking/using-ptp.adoc
+
+[id="cnf-configuring-fifo-priority-scheduling-for-ptp_{context}"]
+= Configuring FIFO priority scheduling for PTP hardware
+
+In telco or other deployment configurations that require low latency performance, PTP daemon threads run in a constrained CPU footprint alongside the rest of the infrastructure components. By default, PTP threads run with the `SCHED_OTHER` policy. Under high load, these threads might not get the scheduling latency they require for error-free operation.
+
+To mitigate against potential scheduling latency errors, you can configure the PTP Operator `linuxptp` services to allow threads to run with a `SCHED_FIFO` policy. If `SCHED_FIFO` is set for a `PtpConfig` CR, then `ptp4l` and `phc2sys` will run in the parent container under `chrt` with a priority set by the `ptpSchedulingPriority` field of the `PtpConfig` CR.
+
+[NOTE]
+====
+Setting `ptpSchedulingPolicy` is optional, and is only required if you are experiencing latency errors.
+====
+
+.Procedure
+
+. Edit the `PtpConfig` CR profile:
++
+[source,terminal]
+----
+$ oc edit PtpConfig -n openshift-ptp
+----
+
+. Change the `ptpSchedulingPolicy` and `ptpSchedulingPriority` fields:
++
+[source,yaml]
+----
+apiVersion: ptp.openshift.io/v1
+kind: PtpConfig
+metadata:
+  name: <ptp_config_name>
+  namespace: openshift-ptp
+...
+spec:
+  profile:
+  - name: "profile1"
+...
+    ptpSchedulingPolicy: SCHED_FIFO <1>
+    ptpSchedulingPriority: 65 <2>
+----
+<1> Scheduling policy for ptp4l and phc2sys processes. Use `SCHED_FIFO` on systems that support FIFO scheduling.
```

Comment:
Throughout the rest of this document, `ptp4l` and `phc2sys` are in backtickets. In this instance (<1>), they are not. Should it be fixed?

Reply:
Yes, will correct.

Reply:
undid my erroneous commit and applied your suggestion, thanks.